Eshop.Domain/Category.cs

```diff
@@ -9,34 +8,23 @@ namespace Eshop.Domain
 {
     public class Category
     {
-        public Category(string itemType) 
-        { 
-
-            string ItemType = itemType;
+        public Category(string categoryName, int id, string description)
+        {
+            this.id = id;
+            this.description = description;
         }
+        public string categoryName { get;  set; }
```

Comment:
1. Why there is prefix Category when you are already in category. When you construct object 
```
var category = new Category()
Console.WriteLine(category.categoryName) // does not make sense
```
You was asked via Code Review to create property Name, so you can not change suggested implementation. Now you waste your time and also mine. You see the point to read carefully and follow the instruction?
2. Why you changed everything to the camelCase - clearly you are not listening or not rewatching videos carefully - params are camelCase, properties are PascalCase. When you have it like this, then you do not need to use for these members this. prefix.
3. Why are you ruining encapsulation. You was not listening correctly again - all setters instead of Id are public - why? If you do not understand why you need to tell me. Because this is very very very very important.

Reply:
I am really sorry for wasting your time but I think I just did not understand what you wanted from me at all - but now 
for 2. I changed it to PascalCase - well I started again
for 3. it is private, so it should not be ruined

Reply:
edit - did not, but it is how it should be